client/client_test.go

```diff
@@ -280,11 +280,13 @@ func testOCIExporter(t *testing.T, sb integration.Sandbox) {
 	defer os.RemoveAll(destDir)
 
 	out := filepath.Join(destDir, "out.tar")
+	target := "docker.io/buildkit/testoci:latest"
```

Comment:
nit: `docker.io/buildkit` is owned by somebody else: https://hub.docker.com/u/buildkit/